datasets/us/co/med_exclusions/crawler.py

```diff
@@ -0,0 +1,99 @@
+from typing import Dict
+from rigour.mime.types import XLSX
+from openpyxl import load_workbook
+from datetime import datetime
+import re
+
+from zavod import Context, helpers as h
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    name = row.pop("provider_name")
+
+    if not name:
+        return
+
+    entity = context.make("LegalEntity")
+    entity.id = context.make_id(name, row.get("npi"))
+    entity.add("name", name)
+
+    if row.get("npi") != "N/A":
+        npis = re.split(r"; |& ", row.pop("npi"))
+
+        for npi in npis:
+            entity.add("npiCode", npi)
```

Comment:
entity.add happily takes lists

```suggestion
        entity.add("npiCode", npis)
```